SU-HSDO/suhumsci
# STN-153: Reset Button

# READY FOR REVIEW

## Summary
This pull request adds the reset button styles now that we know how to make exposed filters display the button without a plugin. Design has been reviewed by @meranicosme.

This PR also includes a design adjust requested by J to darken the input border and dropdown hover colors.

## Need Review By (Date)
02/05

## Urgency
medium

## Steps to Test
1. Run `npm test` to ensure all tests pass
2. Import [horizontal-view-w-exposed-filters.txt](https://github.com/SU-HSDO/suhumsci/files/4160534/horizontal-view-w-exposed-filters.txt)
3. Create a basic page with a view using the horizontal card with list display:
    <img width="276" alt="Screen Shot 2020-02-03 at 12 17 19 PM" src="https://user-images.githubusercontent.com/2486846/73674767-3445c500-467f-11ea-95ab-2d9eabb564db.png">
4. Review that the reset button matches the [design mockup](https://www.figma.com/file/toNZnFLadtpNpG5dPzNGWz/%E2%9A%99%EF%B8%8F-Colorful-Theme---Working?node-id=2067%3A0)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
